var/spack/repos/builtin/packages/R/package.py

```diff
@@ -47,3 +59,45 @@ def install(self, spec, prefix):
         configure(*options)
         make()
         make('install')
+
+    # ========================================================================
+    # Set up environment to make install easy for R extensions.
+    # ========================================================================
+
+    @property
+    def r_lib_dir(self):
+        return os.path.join('lib64', 'R', 'library')
```

Comment:
`lib64` isn't used on Ubuntu or OS X, so this will need some logic to handle other systems.